scripts/azure-pipelines/build.ps1

```diff
@@ -0,0 +1,48 @@
+﻿# Builds Mumble using the given environment and build script.
+# The configuration we build with is adjusted to be close to
+# our release builds.
+#
+# Configuration variables used from environment:
+#
+#  AGENT_BUILDDIRECTORY       - The local path on the agent where all folders
+#                               for a given build pipeline are created
+#                               (e.g. "D:\a\1")
+#  BUILD_SOURCESDIRECTORY     - The local path on the agent where the
+#                               repository is downloaded.
+#                               (e.g. "D:\a\1\s")
```

Comment:
So does this have to be prefixed with the value from `AGENT_BUILDDIRECTORY`?

Reply:
These are predefined build variables: https://docs.microsoft.com/en-us/azure/devops/pipelines/build/variables